code_samples/action_basics/.gitignore

```diff
@@ -0,0 +1,11 @@
+# Created by .ignore support plugin (hsz.mobi)
```

Comment:
we usually do not share .gitignore files in our samples

Reply:
My bad. I thought gradle_plugin_demo had one, so I added it. I see now that gradle_plugin_demo does not. I will remove it.